plexus-compilers/plexus-compiler-eclipse/pom.xml

```diff
@@ -14,7 +14,7 @@
   <description>Eclipse Compiler support for Plexus Compiler component.</description>
 
   <properties>
-    <maven.compiler.release>11</maven.compiler.release>
+    <javaVersion>17</javaVersion>
```

Comment:
Not yet sure why, but this breaks using eclipse compiler. With 17 here I'm getting 
```
No such compiler 'eclipse'.
```
Once set to 11 - ecj works fine again.
Please restore.

Reply:
You need a newer Maven (Sosu) see release notes

Reply:
@slachiewicz Thanks - I've completely missed that in RN.